src/sage/groups/braid.py

```diff
@@ -2239,6 +2240,95 @@ def colored_jones_polynomial(self, N, variab=None, try_inverse=True):
         self._cj_with_q[N] = cj.subs({q: 1/q}) if use_inverse else cj
         return self.colored_jones_polynomial(N, variab, try_inverse)
 
+    def send_to_sss(self):
+        r"""
+        Return an element of the braid's super summit set, an the conjugating
+        braid.
+
+        EXAMPLES::
+
+            sage: B = BraidGroup(4)
+            sage: b = B([1, 2, 1, 2, 3, -1, 2, 1, 3])
+            sage: b.send_to_sss()
+            (s0*s2*s0*s1*s2*s1*s0, s0^-1*s1^-1*s0^-1*s2^-1*s1^-1*s0^-1*s1*s0*s2*s1*s0)
+
+        """
+        to_sss = send_to_sss(self)
+        B = self.parent()
+        return tuple(B._element_from_libbraiding(b) for b in to_sss)
+
+    def send_to_uss(self):
+        r"""
+        Return an element of the braid's ultra summit set, an the conjugating
+        braid.
+
+        EXAMPLES::
+
+            sage: B = BraidGroup(4)
+            sage: b = B([1, 2, 1, 2, 3, -1, 2, -1, 3])
+            sage: b.send_to_uss()
+            (s0*s1*s0*s2*s1, s0^-1*s1^-1*s0^-1*s2^-1*s1^-1*s0^-1*s1*s2*s1^2*s0)
+
+        """
+        to_uss = send_to_uss(self)
+        B = self.parent()
+        return tuple(B._element_from_libbraiding(b) for b in to_uss)
```

Comment:
```suggestion
        return tuple([B._element_from_libbraiding(b) for b in to_uss])
```
For whatever reason, this is marginally faster.